Assets/Scripts/Dialogue/DialogueSystem.cs

```diff
@@ -87,7 +96,7 @@ private IEnumerator DisplayDialogue()
             yield return textbox.Say();
 
             // Wait for the user to advance the dialogue
-            yield return new WaitUntil(() => CheckIfAdvanceKeyPressed() || skip);
+            yield return new WaitUntil(() => CheckIfAdvanceKeyPressed() || skip || advKeyPressed);
```

Comment:
is this redundant? CheckIfAdvanceKeyPressed already checks for advKeyPressed. we can resolve this later.